corpus/literals.txt

```diff
@@ -314,6 +314,50 @@ class A {
           (identifier)
           (equals_value_clause (string_literal (escape_sequence)))))))))
 
+=======================================
+utf-8 string literals
+=======================================
+
+var a = "lower"u8;
+var b = "upper"U8;
+var c = @"lower"u8;
+var d = @"upper"U8;
```

Comment:
A reference would be helpful here. I couldn't find this syntax in previews of the upcoming C# 11 features.

Reply:
https://github.com/dotnet/csharplang/blob/main/proposals/utf8-string-literals.md is where the details can be found. Most of the proposals for C# start in that repo and you can see it was approved in [January 2022](https://github.com/dotnet/csharplang/blob/main/meetings/2022/LDM-2022-01-26.md#suffix-case-sensitivity) and has been available in Visual Studio 2022 Beta for several months.

Reply:
:+1: My search was definitely too shallow to uncover that :)

Reply:
Yeah keeping up with the C# syntax is not a nice experience :( Blog posts, proposals, stuff in and out of VS and Roslyn... I think these two are solid based on the discussion/lack of controversy but could hold off another month.

Reply:
I don't know what usually happened? You said it is already available in VS Beta. If people can write it, that could be an argument to already merge it?